modules/Hoot/ui/managePanel/transAssist/upload.js

```diff
@@ -7,26 +7,16 @@
 export default class Upload {
     constructor( instance ) {
         this.instance = instance;
-
-        this.schemaOptions = [
-            {
-                name: 'OSM',
-                enabled: true,
-                checked: true
-            },
-            {
-                name: 'TDSv70',
-                enabled: true
-            },
-            {
-                name: 'TDSv61',
-                enabled: true
-            },
-            {
-                name: 'MGCP',
+        this.schemaOptions = Hoot.translations.availableTranslations.map(v => {
```

Comment:
can you make `schemaOpts` in this class a drop down instead of a list of radio buttons?